src/include/benchmark/ycsb/ycsb_workload.h

```diff
@@ -41,7 +41,9 @@ std::vector<std::vector<type::Value>> ExecuteRead(executor::AbstractExecutor* ex
 
 void ExecuteUpdate(executor::AbstractExecutor* executor);
 
+#ifndef __APPLE__ // https://superuser.com/questions/149312/how-to-set-processor-affinity-on-os-x
```

Comment:
Can you add a brief instructive comment inside the if-def about why this was done? Include the link for those that want to know more.